src/www/services_dhcpv6.php

```diff
@@ -38,7 +38,7 @@ function reconfigure_dhcpd()
 {
     system_hosts_generate();
     clear_subsystem_dirty('hosts');
-    dhcpd_dhcp_configure(false, 'inet6');
+    dhcpd_dhcp_configure(false); // restart dhcp4
```

Comment:
this makes no sense?

Reply:
Try it, if you make a change and save it in dhcp4 then it leaves dhcp6 stopped.